backend/queryparser/queryparser.go

```diff
@@ -0,0 +1,19 @@
+package queryparser
+
+import "strings"
+
+func Parse(query string) map[string]string {
```

Comment:
I extracted this out into a utility so we can use it for various parts of the app (searching logs, error objects, traces?).

It's not as comprehensive as logs search so I didn't update that code path but I figured we can use this as a starting point on how our query language can be re-used across various products.

Reply:
FYI @ccschmitz @Vadman97 - if we use the same query language for searching trace logs, it would be great to consolidate here.